javascript-loops-practice/add-suffix-to-all.js

```diff
@@ -0,0 +1,10 @@
+/* exported addSuffixToAll */
+
+function addSuffixToAll(words, suffix) {
+  var finalWord = [];
+  for (var i = 0; i < words.length; i++) {
+    words[i] += suffix;
```

Comment:
This is bad news, you don't want to make changes to the original array passed as an argument, unless making permanent changes to the array is the intended purpose of the function. You should just use the value of that index to generate a new value you can add to your output array